src/EditorFeatures/VisualBasic/EndConstructGeneration/EndConstructCommandHandler.vb

```diff
@@ -22,6 +23,7 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.EndConstructGeneration
     <Name(PredefinedCommandHandlerNames.EndConstruct)>
     <Order(After:=PredefinedCommandHandlerNames.Completion)>
     <Order(After:=PredefinedCommandHandlerNames.AutomaticLineEnder)>
+    <Order(After:=PredefinedCompletionNames.CompletionCommandHandler)>
```

Comment:
my preference in all cases would be to pair this wiht the existing 'Completion' order attribute above.